### DISCUSSION TITLE
v0.8.3

### DISCUSSION THREAD

**Discussion by Revertron** [Revertron/Alfis#314] [Releases]
## Bug Fixes & Improvements
* Some people had complaint about DNS resolver in ALFIS, that it is overwhelmed with requests, and at some point it just stops to handle DNS requests at all. For some reason I didn't see the problem in that code of DNS resolver, but today it hit me - the work handling by all that threads is wrong.

So, the bug of DNS resolver is resolved ;)
Now, if you have `threads` set to some high number, like 100 or 200, you can lower that significantly. I recommend the number around your CPU core count. It will suffice.

<hr /><em>This discussion was created from the release <a href='https://github.com/Revertron/Alfis/releases/tag/v0.8.3'>v0.8.3</a>.</em>

**Comment by tyden70** [comment]
Ubuntu 22.02.1 x64 -- не устaнавливается, помечена как потенциально опасная.
Устонавливал через установщик.
На Винде без замечаний.
